tests/cli/test_buildspec.py

```diff
@@ -150,6 +152,24 @@ def test_buildspec_find_invalid():
     cache.print_invalid_buildspecs(error=False, terse=True, header=True)
 
 
+@pytest.mark.cli
+def test_edit_test():
+    edit_buildspec_test(
+        buildspecs=[os.path.join(BUILDTEST_ROOT, "tutorials", "vars.yml")],
```

Comment:
so this method doesn't accept `buildspecs` but instead you should use one of the valid test names. See the method signature, you need to use `test_names` to pass the test name which takes the argument for name of test when you say `buildtest bc edit-test hello_world` the name of test is `hello_world`.

Reply:
i believe `hello_world` is a test name that should exist you can call this method like
```
    edit_buildspec_test(
        test_names=["hello_world",],
        configuration=configuration,
        editor='',
    )
```

Reply:
This was the code snipet that you gave to use in this spot. So instead i should just give the method call a test name like hello world? 
IG: edit_buildspec_test(buildspecs="hello_world",configuration=configuration,editor="")?